website/versioned_docs/version-23.0/MockFunctions.md

```diff
@@ -36,7 +36,7 @@ expect(mockCallback.mock.calls[0][0]).toBe(0);
 expect(mockCallback.mock.calls[1][0]).toBe(1);
 
 // The return value of the first call to the function was 42
-expect(mockCallback.mock.returnValues[0]).toBe(42);
+expect(mockCallback.mock.results[0]).toBe(42);
```

Comment:
I think this would be `.results[0].value` and can you add to the `docs/MockFunctions.md` file as well?

Reply:
Done.